plugins/jobs/src/js/data/JobModel.ts

```diff
@@ -198,6 +212,15 @@ export const resolvers = ({
 
       return responses$.map(response => JobTypeResolver(response));
     }
+  },
+  Mutation: {
+    runJob(
+      _parent = {},
+      args: GeneralArgs,
+      _context = {}
+    ): Observable<JobLink> {
+      return runJob(args.id).map(({ jobId }) => ({ jobId }));
```

Comment:
I'd love to see some kind of check for arguments here (see Query resolvers above)

Reply:
I think your point is super valid, but this code is already on master and not on the scope of this ticket (that does not even touch that code). I am adding a note to the graphql ticket so I look at the args of all resolvers I am implementing.

Reply:
I will look with more care, but i would expect that it would be impossible for you to issue a query without an id to args if the types on graphql require too. So that clause should be a code you save because you typed right.